docs/getting-started/genaichatbot.md

```diff
@@ -120,10 +120,10 @@ databricks secrets list --scope slack
 
 ### 1c. Load the Git repository
 
-1.  Login to [Prophecy](https://app.prophecy.io/metadata/auth/signup)
-2.  Create a new Prophecy Project.
-3.  Load the forked Git repository to the Prophecy Project as shown in this 30-second [video.](https://github.com/prophecy-samples/gen-ai-chatbot-template/assets/3248329/dcdfabaf-4870-421d-9f92-4ab028c5db5a), pointing to your fork at `https://github.com/<your_username>/gen-ai-chatbot-template`
-4.  Connect to your Spark cluster by creating a Fabric following [these steps.](https://docs.prophecy.io/concepts/fabrics/create-a-fabric/#databricks)
+1.  **Login** to [Prophecy](https://app.prophecy.io/metadata/auth/signup)
+2.  **Import** a new Prophecy [Project](/docs/concepts/project/project.md#1-create-new-project)
+3.  **Load the forked Git repository** to the Prophecy Project as shown in this 30-second [video.](https://github.com/prophecy-samples/gen-ai-chatbot-template/assets/3248329/dcdfabaf-4870-421d-9f92-4ab028c5db5a), pointing to your fork at `https://github.com/<your_username>/gen-ai-chatbot-template`
```

Comment:
Just linking to a video isn't 100% accessible. We should have a separate section in `project.md` about importing existing projects instead and we can just link to it.

Reply:
I agree, though I need to move onto other doc tasks. Can revisit when I do this: https://app.asana.com/0/1203457878958265/1205308678114911/f

Reply:
@slyons would you mind approving this PR, with the plan to revisit?